lib/reek/cli/options.rb

```diff
@@ -9,7 +9,7 @@ module CLI
     #
     # See {file:docs/Command-Line-Options.md} for details.
     #
-    # :reek:TooManyInstanceVariables: { max_instance_variables: 9 }
+    # :reek:TooManyInstanceVariables: { max_instance_variables: 10 }
```

Comment:
>> replacing the bunch of instance variables with a struct?

This wouldn't fix the smell, just defer it to somewhere else. 

>>  class variables?

That would actually make it worse since class variables are evil. Not to mention that we actually have a [smell detector](https://github.com/troessner/reek/blob/master/docs/Class-Variable.md) for it :laughing: 

One solution I had in the back of my mind was to use composition, which in this case means to have several different objects encapsulating groups of cli arguments. We could e.g. group / abstract the exit codes in one object, and the format arguments in another. 
Since I find this "put all arguments into one instance annoying and slightly confusing as well I'll probably look into fixing this sooner or later (but in another PR then).